projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-v1-4-endpoint.php

```diff
@@ -117,6 +117,7 @@
 			'launchpad_screen'                        => '(string) Whether or not launchpad is presented and what size it will be',
 			'featured_image_email_enabled'            => '(bool) Whether the Featured image is displayed in the New Post email template or not',
 			'wpcom_gifting_subscription'              => '(bool) Whether gifting is enabled for non auto-renew sites',
+			'jetpack_blogging_prompts_enabled'        => '(bool) Whether writing prompts are shown in the editor when starting a new post.',
```

Comment:
should the 1.3 endpoint be updated too? Maybe not, I can see that calypso uses the 1.4 endpoint.